frontend/src/entry/model/settings/ModelRolesInfo.tsx

```diff
@@ -0,0 +1,76 @@
+import { Box, Grid, Typography } from '@mui/material'
+import { Stack } from '@mui/system'
+import { useGetModelRoles } from 'actions/model'
+import { useCallback } from 'react'
+import Loading from 'src/common/Loading'
+import MessageAlert from 'src/MessageAlert'
+import { RoleKind } from 'types/types'
+
+interface ModelRolesInfoProps {
+  modelId: string
+}
+
+export default function ModelRolesInfo({ modelId }: ModelRolesInfoProps) {
+  const { modelRoles, isModelRolesLoading, isModelRolesError } = useGetModelRoles(modelId)
+
+  const roles = useCallback(
+    (kind: string) => {
+      return modelRoles
+        .filter((modelRole) => modelRole.kind === kind)
+        .map((filteredModelRole) => (
+          <Box key={filteredModelRole.id}>
+            <Typography fontWeight='bold'>{filteredModelRole.name}</Typography>
+            <Typography>{filteredModelRole.description}</Typography>
+          </Box>
+        ))
+    },
+    [modelRoles],
+  )
+
+  if (isModelRolesError) {
+    return <MessageAlert message={isModelRolesError.info.message} severity='error' />
+  }
+
+  return (
+    <>
+      {isModelRolesLoading && <Loading />}
+      {!isModelRolesLoading && (
+        <Stack spacing={2}>
+          <Typography>
+            Roles in Bailo are split into two categories; standard and dynamic. Standard roles are generic across
```

Comment:
This block of text may need to be genericised to use the entry kind, instead of statically referring to "model". Either that or we need to conditionally show a model/data card specific block of text.